src/main/java/cz/muni/fi/pa165/bluebat/entity/Customer.java

```diff
@@ -1,5 +1,95 @@
-package cz.muni.fi.pa165.bluebat.entity;
+package cz.muni.fi.pa165.bluebat.entity;/* created by rudolf */
 
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.util.Objects;
+
+/**
+ * @author rudolf
+ */
+@Entity
 public class Customer {
```

Comment:
Plus by nebylo špatné používat Lombok. Je to jednoduché, pro inspiraci se lze podívat na ostatní pull requesty.